releasenotes/notes/feat:-detect-attack-on-body-3fe45bc066dcc680.yaml

```
@@ -0,0 +1,3 @@
+features:
+  - |
+    Detect attacks on Django body.
```

```suggestion
    asm: Detect attacks on Django body.
```

? wondering if we should all be better about prefixing with the product so people know more context.